### PR TITLE
[126] Move time to confirm to setting

### PR DESCRIPTION
**Pull request by przemyslawhaptas**

### CODE REVIEW

File: app/models/setting.rb [hunk 1]
```diff
@@ -10,15 +10,17 @@ class Setting < ActiveRecord::Base
...
-  validates :required_rates_num, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-
+  validates :available_spots, :required_rates_num, :confirmation_time,
+            numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
   def self.get
     self.first || self.create({
       available_spots: 0,
       required_rates_num: 3,
+      confirmation_time: 7,
```
Comment (takiarek) [inline app/models/setting.rb]: Regarding the comment about naming, an idea popped up in my mind: days_to_confirm_invitation. Although, I'm not sure about it.

File: app/views/settings/_form.html.erb [hunk 2]
```diff
@@ -22,6 +22,10 @@
     <%= f.number_field :required_rates_num, min: 0, required: true %>
   </div>
   <div class="form-field">
+    <%= f.label :confirmation_time %>
+    <%= f.number_field :confirmation_time, min: 0, required: true %>
```
Comment (takiarek) [inline app/views/settings/_form.html.erb]: Users won't know that their setting number of days and not weeks or hours.

File: spec/models/setting_spec.rb [hunk 3]
```diff
@@ -1,21 +1,23 @@
 require "rails_helper"
+require "models/shared/positive_present_integer"
+require "models/shared/present"
```
Comment (takiarek) [inline spec/models/setting_spec.rb]: Maybe it's just me but without reading further I wasn't sure what the word 'present' related to. How about using the word 'presence' to remove ambiguity?

File: app/models/setting.rb (hunk 1 above)
Comment (przemyslawhaptas) [inline app/models/setting.rb]: Well I can't store 7.days in the db and I don't think mapping 7.days (model) to 7 (db) is a good idea. Good naming should be enough, I'll change it to what you proposed.

File: spec/models/setting_spec.rb (hunk 3 above)
Comment (przemyslawhaptas) [inline spec/models/setting_spec.rb]: I'll just merge these two together and name it "validations". ;)

**Comment by przemyslawhaptas** [comment]
[[126] Time to confirm invitation in settings](https://trello.com/c/cqezkhQN/96-126-time-to-confirm-invitation-in-settings)

**Pull request merged**